config/_templates/dataset/yaak/predict.yaml

```diff
@@ -18,40 +18,33 @@
 #@     'G1-00436/2024-04-22--15-59-18'
 #@ ]
 
-#@ cameras = [
```

Comment:
keep for adding more cameras ?

Reply:
ytt templating gets pretty messy, keeping it simpler for now until the need arises